src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -10,66 +15,70 @@ export module DotNet {
-    }
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      constructor(private _jsObject: any) {
+          this._cachedFunctions = new Map<string, Function>();
+      }
 
-    public findFunction(identifier: string) {
-      const cachedFunction = this._cachedFunctions.get(identifier);
+      public findFunction(identifier: string) {
+          const cachedFunction = this._cachedFunctions.get(identifier);
 
-      if (cachedFunction) {
-        return cachedFunction;
-      }
+          if (cachedFunction) {
+              return cachedFunction;
+          }
 
-      let result: any = this._jsObject;
-      let lastSegmentValue: any;
+          let result: any = this._jsObject;
+          let lastSegmentValue: any;
 
-      identifier.split('.').forEach(segment => {
-        if (segment in result) {
-          lastSegmentValue = result;
-          result = result[segment];
-        } else {
-          throw new Error(`Could not find '${identifier}' ('${segment}' was undefined).`);
-        }
-      });
+          identifier.split(".").forEach(segment => {
+              if (segment in result) {
+                  lastSegmentValue = result;
+                  result = result[segment];
+              } else {
+                  throw new Error(`Could not find '${identifier}' ('${segment}' was undefined).`);
+              }
+          });
 
-      if (result instanceof Function) {
-        result = result.bind(lastSegmentValue);
-        this._cachedFunctions.set(identifier, result);
-        return result;
-      } else {
-        throw new Error(`The value '${identifier}' is not a function.`);
+          if (result instanceof Function) {
+              result = result.bind(lastSegmentValue);
+              this._cachedFunctions.set(identifier, result);
+              return result;
+          }
+
+          throw new Error(`The value '${identifier}' is not a function.`);
       }
-    }
 
-    public getWrappedObject() {
-      return this._jsObject;
-    }
+      public getWrappedObject() {
+          return this._jsObject;
+      }
   }
 
   const pendingAsyncCalls: { [id: number]: PendingAsyncCall<any> } = {};
   const windowJSObjectId = 0;
   const cachedJSObjectsById: { [id: number]: JSObject } = {
-    [windowJSObjectId]: new JSObject(window),
+      [windowJSObjectId]: new JSObject(window)
   };
 
-  cachedJSObjectsById[windowJSObjectId]._cachedFunctions.set('import', (url: any) => {
-    // In most cases developers will want to resolve dynamic imports relative to the base HREF.
-    // However since we're the one calling the import keyword, they would be resolved relative to
-    // this framework bundle URL. Fix this by providing an absolute URL.
-    if (typeof url === 'string' && url.startsWith('./')) {
-      url = document.baseURI + url.substr(2);
-    }
+  cachedJSObjectsById[windowJSObjectId]._cachedFunctions.set("import", (url: any) => {
+
+      /*
```

Comment:
Is this the idiomatic way to do multi-line comments in TypeScript? We generally don't do it in C# so if it's not super official, I'd prefer we leave this be.

Reply:
[`starred blocks`](https://eslint.org/docs/rules/multiline-comment-style) are the default multi-line comment configuration for `eslint` so that _may_ be idiomatic. It is configurable though to `separate-lines` so I'll change it back.